css/media.css

```
@@ -0,0 +1,110 @@
+@media (max-width:1080px){
+    .social-network{
+        display: none;
+    }
+}
+
+@media (min-width: 1200px) and (max-width: 1483px) {
+    .swiper-button--prev {
+          margin-left: 10%; 
+    }
+    .swiper-button--next {
+          margin-right: 10%; 
+    } 
+}
+
+@media (max-width:768px){
```

```suggestion
@media (max-width:767px){
```

так как планшет начинается с 768 включительно

---

Выполнил